app/furniture/marketplace/marketplace_component.html.erb

```diff
@@ -1,9 +1,8 @@
 <div class="grid grid-cols-1 gap-6">
-
-
+  <%- if hero_image.present? %>
+    <%= image_tag hero_image, class: "rounded-lg w-full" %>
+  <%- end %>
```

Comment:
Ah, interesting choice to include it on the `Marketplace`, rather than the `Section`. I was presuming that the `Section#hero_image` would apply to any `Section`, not just ones that use the `Marketplace`.

If you're down for moving this up to the `Section` level I would be stoked; but I think this is fine as is.

Reply:
Ha, I'm embarrassed because I meant to do that. How about now? I'm just not sure I love the design because the image is so big at the top.

Reply:
Now it looks like it's in the iterator over ever `Gizmo`. What about putting it in `sections/_section.html.erb`  or even `sections/show.html.erb`? 

And yea, being a Hero image we would want it to be full-width; which may be intense? But maybe it's fine?

Reply:
I think maybe the section Hero Image the the Card Hero Image want to be different aspect ratios; perhaps the Section Hero Image never exceeds say, 420px high; and is as wide as the viewport?

Reply:
We could definitely use some design review!